src/commands/configuration/FeedbackChannel.ts

```diff
@@ -0,0 +1,206 @@
+                .addSubcommand((subcommand) =>
+                    subcommand
+                        .setName("remove")
+                        .setDescription("Remove a feedback channel")
+                )
+                .setDMPermission(false)
+                .setDefaultMemberPermissions(PermissionFlagsBits.ManageGuild)
+        );
+    }
+
+    async execute(
+        client: DiscordClient<true>,
+        interaction: DiscordChatInputCommandInteraction<"cached">
+    ) {
+
+        switch (interaction.options.getSubcommand()) {
+            case "add": {
+
+                const channel = interaction.options.getChannel("channel", true);
+                const label = interaction.options.getString("team_label", true);
+
+                if (!channel?.isTextBased() || !channel) {
+                    await interaction.reply({
+                        content: "Feedback channel must be a valid text channel.",
+                        ephemeral: true
+                    })
+                    return;
+                }
+
+
+                if (!label) {
+                    await interaction.reply({
+                        content: "Please enter a valid team label.",
+                        ephemeral: true
+                    })
+                    return;
+                }
+
+                await interaction.reply({
+                    content: `Adding feedback channel ${channel} for ${label}`,
+                    ephemeral: true
+                })
+
+                try {
+                    const addChannel = await FeedbackChannels.updateOne(
+                        {
+                            guildId: interaction.guildId,
+                            label: label
+                        },
+                        {
+                            channelId: channel.id
+                        },
+                        { upsert: true }
+                    );
+
+                    addChannel.upsertedCount > 0 ?
+                        await interaction.editReply({
+                            content: `Successfully added ${channel} as the feedback channel for ${label}!`
+                        }) :
+                        addChannel.modifiedCount > 0 ?
+                            await interaction.editReply({
+                                content: `Successfully updated ${channel} as the feedback channel for ${label}`
+                            }) :
+                            await interaction.editReply({
+                                content: `Error occured while creating feedback channel ${channel} for ${label}. Please try again later.`
+                            });
+                } catch (error) {
+                    await interaction.editReply({
+                        content:
+                            `Encountered error while trying to add ${channel} as a feedback channel for ${label}. Please try again later.`
+                    });
+
+                    console.log(error);
```

Comment:
Use `Logger.error`

Reply:
Uncommenting the lines after, I can safely remove this line since we're logging the error via `client.log` right?